gr-digital/lib/costas_loop_cc_impl.cc

```diff
@@ -167,15 +77,24 @@ int costas_loop_cc_impl::work(int noutput_items,
     float* phase_optr = output_items.size() >= 3 ? (float*)output_items[2] : NULL;
     float* error_optr = output_items.size() >= 4 ? (float*)output_items[3] : NULL;
 
-    gr_complex nco_out;
-
     std::vector<tag_t> tags;
     get_tags_in_range(tags,
                       0,
                       nitems_read(0),
                       nitems_read(0) + noutput_items,
                       pmt::intern("phase_est"));
 
+    // Get this out of the for loop if not used:
+    bool hasAdditionalOutputs = false;
```

Comment:
``hasAdditionalOutputs => has_additional_outputs``. CamelCase is not used in GNU Radio’s code as far as I know.

Reply:
+1 here from me too

Reply:
:camel: